AUTHORS

```diff
@@ -2316,5 +2316,378 @@ Ben Newman <ben@meteor.com>
+Nikhil M <m_nikhil@outlook.com>
+leeight <leeight@gmail.com>
+Selvaraj <selvarajrch@gmail.com>
+Mayank Asthana <mayankasthana1993@gmail.com>
+Simon Bruce <simon@blacksun.cx>
+NoSkillGirl <singhpooja240393@gmail.com>
+apoorvanand <apoorwanand@gmail.com>
+Jay Arthanareeswaran <jarthana@in.ibm.com>
+Amanpreet <amanpreet6309108@gmail.com>
+kanishk30 <kanishk71@gmail.com>
+UjjwalUpadhyay <ujjwalkool1990@gmail.com>
+Namit Bhalla <oyenamit@gmail.com>
+Matteo <teorossi82@yahoo.it>
+dexterleng <thequicklab@gmail.com>
+Kunda Sunil Kumar <kunda.sunilkumar@gmail.com>
+rahulshuklab4u <rahulshuklab4u@gmail.com>
+gengjiawen <technicalcute@gmail.com>
+Maya Anilson <manilson@yahoo.in>
+Mrityunjoy Saha <msaha@vmware.com>
+Robin Drexler <drexler.robin@gmail.com>
+Prabu Subra <prabus69@gmail.com>
+Abhishek Dixit <abhishek.dixit@genisys-group.com>
+Sarath Govind K K <sarathgovind27@gmail.com>
+Dexter Leng <dexterlengcs@gmail.com>
+Gary Hsu <bghgary@users.noreply.github.com>
+Dan Foley <dfoley@cantremember.com>
+Pranay Kothapalli <pranaykothapalli@gmail.com>
+Nethra Ravindran <nethra.ravindran@in.ibm.com>
+Juan José Arboleda <soyjuanarbol@gmail.com>
+prodroy1 <prodroy1@in.ibm.com>
+Yusuke Kawasaki <u-suke@kawa.net>
+yuriettys <yres.101@gmail.com>
+sadness_ojisan <sadness.ojisan@gmail.com>
+ka2jun8 <narchbald103@gmail.com>
+sota1235 <sota1235@gmail.com>
+dominikeinkemmer <dominik_einkemmer@gmx.at>
+mki-skt <sakito21@gmail.com>
+Laszlo.Moczo <laszlo.moczo@hu.ibm.com>
+jamesgeorge007 <jamesgeorge998001@gmail.com>
+koki-oshima <koki.oshima@weblio.co.jp>
+takato <taishi.k0903@gmail.com>
+yosuke ota <otameshiyo23@gmail.com>
+tottokotkd <tottokotkd@me.com>
+kazuya kawaguchi <kawakazu80@gmail.com>
+Naojirou Hisada <naojirouhisada@gmail.com>
+tpanthera <bisht11390@gmail.com>
+feng jianmei <fengjianmei@fengs-MacBook-Pro.local>
+DoiChris <pej.deutinger@gmail.com>
+VeysonD <veysond@mercari.com>
+susantruong <susantruong2412@gmail.com>
+Takahiro Nakamura <dxbxdxbxdxb+dev@gmail.com>
+Daiki Arai <darai@yahoo-corp.jp>
+horihiro <hori.kawa.shima.tool+github@gmail.com>
+grimrose <grimrose@users.noreply.github.com>
+timothy searcy <timmyzsearcy@gmail.com>
+nakashima <nakashima1711@gmail.com>
+/Jesse <jesse.katsumata@gmail.com>
+Tadhg Creedon <tadhgcreedon@gmail.com>
+exoego <mogami@exoego.net>
+sigwyg <sigwyg>
+pastak <pasta0915@gmail.com>
+lakatostamas <mail.tamaslakatos@gmail.com>
+Mike Samuel <mikesamuel@gmail.com>
+Suguru Motegi <suguru.motegi@gmail.com>
+Gerhard Stoebich <18708370+Flarna@users.noreply.github.com>
+jy95 <jacques.yakoub@gmail.com>
+Kenigbolo Meya Stephen <expensivestephen@hotmail.com>
+Charles Samborski <demurgos@demurgos.net>
+zhmushan <zhmushan@qq.com>
+yoshimoto koki <yoyoyo427435@gmail.com>
+Ilarion Halushka <hilarion.galushka@gmail.com>
+Yael Hermon <yaelhe@wix.com>
+Mitch Hankins <mitchhankins92@gmail.com>
+Mikko Rantanen <jubjub@jubjubnest.net>
+wenjun ye <1728914873@qq.com>
+briete <briete5814@gmail.com>
+Matteo Collina <hello@matteocollina.com>
```

Comment:
I think this is @mcollina?

Reply:
Yes.

Reply:
This was happening because we didn’t have support for `.mailmap` entries for `Co-authored-by:` or `Author:` tags that were manually added to the commit message.

8f2bb03609c3b4406f80dc4f71d91bdd27ac73f8 should remedy that.